src/redux/actions/samples/createSample.js

```diff
@@ -35,14 +49,22 @@ const createSample = (
       .reduce((acc, curr) => ({ ...acc, [curr]: DEFAULT_NA }), {}) || {},
   };
 
-  dispatch({
-    type: SAMPLES_SAVING,
-    payload: {
-      message: endUserMessages.SAVING_SAMPLE,
-    },
-  });
+  if (config.currentApiVersion === api.V1) {
+    url = `/v1/projects/${projectUuid}/${experimentId}/samples`;
+
+    body = _.clone(newSample);
+  } else if (config.currentApiVersion === api.V2) {
+    url = `/v2/experiments/${experimentId}/samples/${newSampleUuid}`;
 
-  const url = `/v1/projects/${projectUuid}/${experimentId}/samples`;
+    let sampleTechnology;
+    if (type === '10X Chromium') {
+      sampleTechnology = '10x';
+    } else {
+      throw new Error(`Sample type ${type} is not implemented`);
+    }
+
+    body = { name, sampleTechnology };
```

Comment:
Why are we not adding the rest of the items in `sampleTemplate` to the body? I see that we are adding them only for v1

Reply:
Because we don't need any of them for api v2:
- `experimentId` is being passed in the url
- `sampleId` is being passed in the url
- createdDate (createdAt) and lastModified (updatedAt) are managed by the api
- metadata is managed by the api in `MetadataTrack`'s method `createNewSampleValues`